opal/mca/pmix/pmix112/pmix/include/pmix/autogen/pmix_config_bottom.h

```diff
@@ -343,26 +343,10 @@
 #    define __pmix_attribute_destructor__
 #endif
 
-#ifndef PMIX_DECLSPEC
-# define PMIX_DECLSPEC
-#ifdef PMIX_C_HAVE_VISIBILITY
-# if PMIX_C_HAVE_VISIBILITY
-#  define PMIX_EXPORT __attribute__((__visibility__("default")))
-# else
-#  define PMIX_EXPORT
-# endif
-#else
-# define PMIX_EXPORT
-#endif
-#endif
-
-#ifndef PMIX_DECLSPEC
-#define PMIX_DECLSPEC
-#  if PMIX_C_HAVE_VISIBILITY
+#if PMIX_C_HAVE_VISIBILITY
```

Comment:
Minor nit: But I would add the redefinition protection before this out of "abundance of caution".
```diff
+ #ifndef PMIX_EXPORT
  #if PMIX_C_HAVE_VISIBILITY
  #    define PMIX_EXPORT           __pmix_attribute_visibility__("default")
  #else
  #    define PMIX_EXPORT
  #endif
+ #endif
```

It's minor enough that I'm not willing to hold up approval for this PR.